packages/celotool/src/e2e-tests/validator_order_tests.ts

```diff
@@ -0,0 +1,109 @@
+import { ContractKit, newKit } from '@celo/contractkit'
+import { assert } from 'chai'
+import _ from 'lodash'
+import { getContext, GethTestConfig, sleep } from './utils'
+
+const validatorAddress = '0x47e172f6cfb6c7d01c1574fa3e2be7cc73269d95'
+// @ts-ignore
```

Comment:
Does this need a `ts-ignore`? I would guess it would automatically infer type `string[]`